webdriver-spec.html

```diff
@@ -2547,6 +2547,66 @@ <h3>Processing Capabilities</h3>
 </ol>
 
 </section> <!-- /Processing Capabilities -->
+
+<aside class=example>
+  <p>Capabilities are meant for defining a failsafe matrix selection of a
+    browser configuration in a distributed WebDriver environment. When
+    communicating directly with an endpoint node (i.e. geckodriver),
+    <a>capabilities</a> dictionaries are only used for carrying
+    browser-specific configuration (i.e. <var>moz:firefoxOptions</var>);
+    however, when talking to an <a>intermediary node</a> multiplexer,
+    the request will likely be forwarded to another node: <a>capabilities</a>
+    are used for selecting a node in this network that matches the capability
+    requirements.</p>
+
+    <p>If there is a requirement to have a Firefox webdriver, with no specific
+      requirements in terms of operating system, the requirement object
+      would be:</p>
+
+<pre>{
+  "alwaysMatch": {
+    "browserName": "firefox"
+  }
+}</pre>
+
+    <p>If Firefosx is still a requirement, but the operating system is also
+      to be taken into account and it must be either <code>linux</code> or
+      <code>windows</code>, the requirement object would be:
```

Comment:
I would skip “If Firefosx is still a requirement” and instead say:

> If the operating system is an additional requirement, and it must be either Linux or Windows, this can be defined in the `firstMatch` array, which will be matched iteratively in the defined order: